extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -1253,6 +1277,14 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
     }
   };
 
+  private setCorrectPaddingForInputContainer = () => {
+    if (this.addedRecipients.some(r => r.sendingType === 'cc')) {
+      $('#input-container-cc').css('padding-bottom', this.inputContainerPaddingBottom);
+    } else if (this.addedRecipients.some(r => r.sendingType === 'to')) {
+      $('#input-container-to').css('padding-bottom', this.inputContainerPaddingBottom);
+    }
```

Comment:
here code is almost the same for both cases, what about simplifying it to

```ts
for (const sendingType of ['to', 'cc']) {
      if (this.addedRecipients.some(r => r.sendingType === sendingType)) {
        $(`#input-container-${sendingType}`).css('padding-bottom', this.inputContainerPaddingBottom);
      }
    }
```

Reply:
Sounds good 👍